webview.h

```diff
@@ -1425,6 +1431,38 @@ inline bool enable_dpi_awareness() {
   return true;
 }
 
+inline bool enable_non_client_dpi_scaling(HWND window) {
+  auto user32 = native_library(L"user32.dll");
+  if (auto fn = user32.get(user32_symbols::EnableNonClientDpiScaling)) {
+    return !!fn(window);
+  }
+  return true;
+}
+
+template <typename T> class dpi_scale_t {
+public:
+  dpi_scale_t(T numerator, T denominator)
+      : m_numerator(numerator), m_denominator(denominator) {}
+
+  constexpr T apply_to(T value) {
+    return ::MulDiv(value, m_numerator, m_denominator);
```

Comment:
The [sample for WM_DPICHANGED](https://learn.microsoft.com/en-us/windows/win32/hidpi/wm-dpichanged) uses `MulDiv`.